src/pymortests/models/__init__.py

```diff
@@ -0,0 +1,4 @@
+# This file is part of the pyMOR project (https://www.pymor.org).
+# Copyright pyMOR developers and contributors. All rights reserved.
+# License: BSD 2-Clause License (https://opensource.org/licenses/BSD-2-Clause)
```

Comment:
There's also `pymortests.model`. I'm fine with opening a subpackage for this, but then these tests should be moved as well. Otherwise, just add the tests to this file for now.